### DISCUSSION TITLE
Jetbrains Compose / Kotlin Multiplatform

### DISCUSSION THREAD

**Discussion by marcguilera** [bumble-tech/appyx#295] [Ideas]
I am just getting started with Appyx so bear with me if I say something that's wrong or incomplete. 

Would it be possible to use this library on Jetbrains Compose? 

From my initial inspection it uses core compose features (ui package) which would already be supported. It relies on some JVM parts such as UUID but that could be solved.

This would make this lib usable for basically every platform (iOS is in the pipeline too) through KMM.

Thanks!

**Comment by marcguilera** [comment]
Duplicate: https://github.com/bumble-tech/appyx/discussions/260

**Comment by zsoltk** [comment]
Closing as duplicate.
